modules/private-shared-lib/src/events/outbound_domain_event_message/party_info_callback_received.ts

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+import { DomainEvent } from '../domain_event';
+import { IMessageHeader } from '@mojaloop/platform-shared-lib-messaging-types-lib';
+import { IPartyResult } from '../../types';
+
+export interface IPartyInfoCallbackReceivedMessageData {
+    key: string;
+    partyResult: IPartyResult;
+    timestamp: number | null;
+    headers: IMessageHeader[] | null;
+}
+
+export class PartyInfoCallbackReceivedMessage extends DomainEvent {
+    constructor(data: IPartyInfoCallbackReceivedMessageData) {
+        super({
+            key: data.key,
+            content: data.partyResult,
+            timestamp: data.timestamp,
+            headers: data.headers,
+            name: PartyInfoCallbackReceivedMessage.name,
+        });
+    }
+
+    getBulkId() {
+        return this.getKey().split('_')[0];
```

Comment:
why do we need split here?

Reply:
Good catch. Actually this logic was refactored already. This file is unnecessary, deleting this file.